awx/ui/client/features/output/details.component.js

```diff
@@ -871,7 +871,8 @@ function JobDetailsController (
             instanceGroup,
             environment,
             artifacts,
-            executionNode
+            executionNode,
+            resultTraceback
```

Comment:
This was always returning an undefined. The reason it worked anyway is because this particular module happens to already have reference to the job model, which the `getResultTracebackDetails` function is using implicitly. :joy: 

Other components can and do subscribe to the messages from this service so I went and added the usual boilerplate there just so this data is actually getting sent.